### ISSUE TITLE
IIS bindings without port numbers causes exception on UI load

### ISSUE THREAD

**Issue by akiller** [webprofusion/certify#106]
Hi,

Named pipes and MSMQ bindings don't have a port number associated. When the GUI loads it crashes with an exception inside IISManager.GetSiteBinding() due to trying to read the Port property of a null EndPoint property.

If I remove the bindings the GUI loads as expected.

![bindings](https://cloud.githubusercontent.com/assets/63985/26586744/081bf0d4-4548-11e7-8f1e-102032a0d2fd.png)

```
An error occurred: System.NullReferenceException: Object reference not set to an instance of an object.
   at Certify.Management.IISManager.GetSiteBinding(Site site, Binding binding)
   at Certify.Management.IISManager.GetSiteBindingList(Boolean ignoreStoppedSites, String siteId)
   at Certify.Management.CertifyManager.ImportManagedSitesFromVault(Boolean mergeSitesAsSan)
   at Certify.UI.ViewModel.AppModel.PreviewImport(Boolean sanMergeMode)
   at Certify.UI.ViewModel.AppModel.LoadSettings()
   at Certify.UI.App.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at Certify.UI.App.Main()
```

**Comment by webprofusion-chrisc** [comment]
@akiller thanks very much for pointing this out Andrew, will get a fix in the next release.

**Comment by Marcus-L** [comment]
I just checked and this is fixed in the current version (2.0.13).
